mycroft/util/format.py

```diff
@@ -20,79 +20,38 @@
 The focus of these formatting functions is to create natural sounding speech
 and allow localization.
 """
-import json
-import os
 import datetime
-import re
 import warnings
 
-from collections import namedtuple
 from calendar import leapdays
 from enum import Enum
 
-from mycroft.util.lang import get_full_lang_code, get_primary_lang_code
-
-from mycroft.util.lang.format_en import *
-from mycroft.util.lang.format_pt import *
-from mycroft.util.lang.format_it import *
-from mycroft.util.lang.format_sv import *
-from mycroft.util.lang.format_hu import *
-
-from mycroft.util.lang.format_es import nice_number_es
-from mycroft.util.lang.format_es import nice_time_es
-from mycroft.util.lang.format_es import pronounce_number_es
-from mycroft.util.lang.format_de import nice_number_de
-from mycroft.util.lang.format_de import nice_time_de
-from mycroft.util.lang.format_de import pronounce_number_de
-from mycroft.util.lang.format_fr import nice_number_fr
-from mycroft.util.lang.format_fr import nice_time_fr
-from mycroft.util.lang.format_fr import pronounce_number_fr
-from mycroft.util.lang.format_nl import nice_time_nl
-from mycroft.util.lang.format_nl import pronounce_number_nl
-from mycroft.util.lang.format_nl import nice_number_nl
-from mycroft.util.lang.format_da import nice_number_da
-from mycroft.util.lang.format_da import nice_time_da
-from mycroft.util.lang.format_da import pronounce_number_da
+from lingua_franca.format import (NUMBER_TUPLE, DateTimeFormat, join_list,
+                                  date_time_format, expand_options,
+                                  _translate_word)
+
+from lingua_franca.lang import get_full_lang_code, get_primary_lang_code
+
+from lingua_franca.lang.format_en import *
```

Comment:
This pattern concerns me.  The wildcard imports are a Python anti pattern.  Also, this import list can get unwieldy as we add more language support.  Why not have nice_number, nice_time and pronounce_number functions that take a language parameter?

Reply:
This is exactly what the file you're in does.

It is kept like this for backwards compatibility so it exports the expected symbols. In future releases this file will probably remain just importing the language-agnostic functions from lingua-franca.

Reply:
Ok, please add a comment or TODO indicating this.